### PR TITLE
Create toppo.ms

### PR DESCRIPTION
**Pull request by ZeEnforcer**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            'speech': "You cannot beat a destrution GOD!!",
+            'powers': array(
+                "/buff",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: missing a comma

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                "/destroyworld"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: missing a comma

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true)",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: extra paranthesis for some reason

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms (hunk 1 above)
Comment (ZeEnforcer) [reply CommandHelper/LocalPackages/commands/buffs/toppo.ms]: where?

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
+                "/shot give ". @plr ." destructionbeam",
+                "/shot give ". @plr ." kiblast"
+            ),
+            'alts': array(
+                'toppo'
+            )
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: why did you add a transformation to transform him into toppo? he's already toppo, no transformation needed

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            'val': 'rCbYbUC0L7jPINP8S6CNfXD79bAR/iZ5uLQLSWJM0mfN5mqoWm7Z57ZpOa0p6ekS2BiQ3pn08sT5yk7soExfZz05vtFAywgW5se6cP+wo4lQ6cRiLdvCSFNmYsnJkfcJjPzwlOnLR8euSnB9UzcS+Br00YHhLMoOwE4RuzfShEbL6xG89XEcE/7AMboU820PvaKAFDUdV6ymjYlcpG/FiwVw+avPNw15svesqkqniab6zphZvXKcQ9YvcqoNvnV3vVhsegGpcALpLeCQafp23IfLv3t2bQHE8EpkBfkv/4cubNCjEC9SYqQeDLN7x/5LpNw4A8m1xYuNh/9YrYnLxUwmZVBaGw+rqLEdYyAvwiDatTKSd5SzT2vnvMEv7eF5aclvN6sDWv8/28xtueWcuCqPZyiirxDNbgVIb8ho2iaYUvFTtn0wAwmuK55KMcXuxVtIyK+tUDtGcyqKS6+bPr7ITcQqbJn+ArGoL+IPLS6qA7mkB0oZvGOEQdp9sTv6smWR6XsZVNeccMFj62qdTsN742msdPv9QQ29BhOPtZoyFaXVRClhKH/6vob7vd5Q5/NzRNtyGBVWZlbTwgExtfcfx2EnWAZThLCczCNCVGG8gruluCDEQazoewtNKoBYmpQu7+CeUdshLe0rNjhSZkO5Gij+ri34SfgY+WZdE8I='
+            'trail': '',
+            'op': 'false',
+            'speech': "You cannot beat a destrution GOD!!",
+            'powers': array(
+                "/buff",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: please add < and > around 0-9 like <0-9>

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 6]
```diff
@@ -0,0 +1,37 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§4T§co§epp§co',
+            'class': 'toppo',
+            'sig': 'eyJ0aW1lc3RhbXAiOjE1MTgyNzU0ODI1MDIsInByb2ZpbGVJZCI6IjA4ODBlY2NjZmVlZDRhMmY5ZTIyZDhmNzIxZjM5MTNlIiwicHJvZmlsZU5hbWUiOiJaZUVuZm9yY2VyIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83ZDU4ZmNhYzM5YjY4MzYwNTkzMTNjMTMzYTg0MjIxZmMyYmExN2VhMzZmZjM3ZjdlZTMzOWZjMDUyZDU0NyJ9fX0='
+            'val': 'rCbYbUC0L7jPINP8S6CNfXD79bAR/iZ5uLQLSWJM0mfN5mqoWm7Z57ZpOa0p6ekS2BiQ3pn08sT5yk7soExfZz05vtFAywgW5se6cP+wo4lQ6cRiLdvCSFNmYsnJkfcJjPzwlOnLR8euSnB9UzcS+Br00YHhLMoOwE4RuzfShEbL6xG89XEcE/7AMboU820PvaKAFDUdV6ymjYlcpG/FiwVw+avPNw15svesqkqniab6zphZvXKcQ9YvcqoNvnV3vVhsegGpcALpLeCQafp23IfLv3t2bQHE8EpkBfkv/4cubNCjEC9SYqQeDLN7x/5LpNw4A8m1xYuNh/9YrYnLxUwmZVBaGw+rqLEdYyAvwiDatTKSd5SzT2vnvMEv7eF5aclvN6sDWv8/28xtueWcuCqPZyiirxDNbgVIb8ho2iaYUvFTtn0wAwmuK55KMcXuxVtIyK+tUDtGcyqKS6+bPr7ITcQqbJn+ArGoL+IPLS6qA7mkB0oZvGOEQdp9sTv6smWR6XsZVNeccMFj62qdTsN742msdPv9QQ29BhOPtZoyFaXVRClhKH/6vob7vd5Q5/NzRNtyGBVWZlbTwgExtfcfx2EnWAZThLCczCNCVGG8gruluCDEQazoewtNKoBYmpQu7+CeUdshLe0rNjhSZkO5Gij+ri34SfgY+WZdE8I='
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: the sig and val look reversed

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 7]
```diff
@@ -0,0 +1,36 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§4T§co§ep§4p§co',
+            'class': 'toppo',
+            'sig': 'eyJ0aW1lc3RhbXAiOjE1MTgyNzU0ODI1MDIsInByb2ZpbGVJZCI6IjA4ODBlY2NjZmVlZDRhMmY5ZTIyZDhmNzIxZjM5MTNlIiwicHJvZmlsZU5hbWUiOiJaZUVuZm9yY2VyIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83ZDU4ZmNhYzM5YjY4MzYwNTkzMTNjMTMzYTg0MjIxZmMyYmExN2VhMzZmZjM3ZjdlZTMzOWZjMDUyZDU0NyJ9fX0='
+            'val': 'rCbYbUC0L7jPINP8S6CNfXD79bAR/iZ5uLQLSWJM0mfN5mqoWm7Z57ZpOa0p6ekS2BiQ3pn08sT5yk7soExfZz05vtFAywgW5se6cP+wo4lQ6cRiLdvCSFNmYsnJkfcJjPzwlOnLR8euSnB9UzcS+Br00YHhLMoOwE4RuzfShEbL6xG89XEcE/7AMboU820PvaKAFDUdV6ymjYlcpG/FiwVw+avPNw15svesqkqniab6zphZvXKcQ9YvcqoNvnV3vVhsegGpcALpLeCQafp23IfLv3t2bQHE8EpkBfkv/4cubNCjEC9SYqQeDLN7x/5LpNw4A8m1xYuNh/9YrYnLxUwmZVBaGw+rqLEdYyAvwiDatTKSd5SzT2vnvMEv7eF5aclvN6sDWv8/28xtueWcuCqPZyiirxDNbgVIb8ho2iaYUvFTtn0wAwmuK55KMcXuxVtIyK+tUDtGcyqKS6+bPr7ITcQqbJn+ArGoL+IPLS6qA7mkB0oZvGOEQdp9sTv6smWR6XsZVNeccMFj62qdTsN742msdPv9QQ29BhOPtZoyFaXVRClhKH/6vob7vd5Q5/NzRNtyGBVWZlbTwgExtfcfx2EnWAZThLCczCNCVGG8gruluCDEQazoewtNKoBYmpQu7+CeUdshLe0rNjhSZkO5Gij+ri34SfgY+WZdE8I='
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: the sign and val still look reversed

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+                "/haste <0-9>",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true)",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: too many )

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
+                "/shot give ". @plr ." destructionbeam",
+                "/shot give ". @plr ." kiblast"
+            ),
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: Delete comma and add another right paranthesis

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
+                "/shot give ". @plr ." destructionbeam",
+                "/shot give ". @plr ." kiblast"
+            )
+            (
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: you added a left paranthesis, you needed to add a right paranthesis

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
+                "/shot give ". @plr ." destructionbeam",
+                "/shot give ". @plr ." kiblast"
+            (
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: now you need to delete the left paranthesis and add two right paranthesis

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+                "/jump <0-9>",
+                "/run <0-9>",
+                "/haste <0-9>",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ((,
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: Now you switched these ones instead of the other ones. delete these two left paranthesis and replace with a right paranthesis

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+            'sig': 'rCbYbUC0L7jPINP8S6CNfXD79bAR/iZ5uLQLSWJM0mfN5mqoWm7Z57ZpOa0p6ekS2BiQ3pn08sT5yk7soExfZz05vtFAywgW5se6cP+wo4lQ6cRiLdvCSFNmYsnJkfcJjPzwlOnLR8euSnB9UzcS+Br00YHhLMoOwE4RuzfShEbL6xG89XEcE/7AMboU820PvaKAFDUdV6ymjYlcpG/FiwVw+avPNw15svesqkqniab6zphZvXKcQ9YvcqoNvnV3vVhsegGpcALpLeCQafp23IfLv3t2bQHE8EpkBfkv/4cubNCjEC9SYqQeDLN7x/5LpNw4A8m1xYuNh/9YrYnLxUwmZVBaGw+rqLEdYyAvwiDatTKSd5SzT2vnvMEv7eF5aclvN6sDWv8/28xtueWcuCqPZyiirxDNbgVIb8ho2iaYUvFTtn0wAwmuK55KMcXuxVtIyK+tUDtGcyqKS6+bPr7ITcQqbJn+ArGoL+IPLS6qA7mkB0oZvGOEQdp9sTv6smWR6XsZVNeccMFj62qdTsN742msdPv9QQ29BhOPtZoyFaXVRClhKH/6vob7vd5Q5/NzRNtyGBVWZlbTwgExtfcfx2EnWAZThLCczCNCVGG8gruluCDEQazoewtNKoBYmpQu7+CeUdshLe0rNjhSZkO5Gij+ri34SfgY+WZdE8I='
+            'val': 'eyJ0aW1lc3RhbXAiOjE1MTgyODU5ODU5NDUsInByb2ZpbGVJZCI6IjA4ODBlY2NjZmVlZDRhMmY5ZTIyZDhmNzIxZjM5MTNlIiwicHJvZmlsZU5hbWUiOiJaZUVuZm9yY2VyIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9hYzVlNmY4ZTRjZmQyOGEyNTgxOWE1OWJkYzFjMGYzODRmNWI2MWM5OTk1OTdjZWNiNTY3ZTgxYzMxYzhkZTMifX19'
+            'trail': '',
+            'op': 'false',
+            'speech': "You cannot beat a destrution GOD!!",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: spelled destruction wrong

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+            'powers': array(
+                "/buff",
+                "/jump <0-9>",
+                "/run <0-9>",
+                "/haste <0-9>",
+                "/flyspeed <number>",
+                "/fly",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: need proof of his flying

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: need proof of him destroying a world

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+                "Anything you attack, dies"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: need proof of his hakai

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: need proof of his ability to breathe in space or water

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: Proof of invincibility to fire

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: Need proof of his invincibility

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: need proof of his regeneration

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+                "/haste <0-9>",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: Need proof he is stronger than goku

File: CommandHelper/LocalPackages/commands/buffs/toppo.ms [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+            'op': 'false',
+            'speech': "You cannot beat a destruction GOD!!",
+            'powers': array(
+                "/buff",
+                "/jump <0-9>",
+                "/run <0-9>",
+                "/haste <0-9>",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "Anything you attack, dies"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/toppo.ms]: he doesn't have powers.onepunch
